dev/ts/shared/internal-helpers.ts

```diff
@@ -16,34 +16,33 @@ limitations under the License.
 
 import { log } from './shared-helpers';
 
-export function _log(header: string, message: unknown, ...optionalParams: unknown[]): void {
+export function _log(header: string, message: string, ...optionalParams: unknown[]): void {
```

Comment:
**question:** does that means that unknown is not legit here? it is used as a better alternative as any.
See also my remark about the usage of `logErrorAndOpenAlert` in _dev/ts/component/DropFileUserInterface.ts_

Reply:
### With the recommended-type-checked  
We have the error `ESLint: Invalid type "unknown" of template literal expression.(@typescript-eslint/restrict-template-expressions)` with `console.info(`${header} ${message}`, ...optionalParams);`.

We have the error `ESLint: Invalid operand for a '+' operation. Operands must each be a number or string, allowing a string + any of: `any`, `boolean`, `null`, `RegExp`, `undefined`. Got `unknown`.(@typescript-eslint/restrict-plus-operands)` with `console.info(header + ' ' + message, ...optionalParams);`.

Reply:
This is fine now as some extra changes have been reverted in the code calling this function.